app/qml/ActiveLayerPanel.qml

```diff
@@ -22,6 +42,22 @@ Drawer {
         color: InputStyle.clrPanelMain
     }
 
+    Item {
+        id: stateManager
+        states: [
+            State {
+                name: "setup"
+                PropertyChanges { target: layerPanel; title: "Default survey layer"}
+            }
+
+           ,State {
```

Comment:
Just for better aesthetic impression - can we have the comma right after the closing brace above? :)